packages/configure/index.js

```diff
@@ -0,0 +1,22 @@
+var compose = require('@stamp/compose');
+var Privatize = require('@stamp/privatize');
+
+function configure(_, options) {
+  var configuration = options.stamp.compose.configuration;
+  var deepConfiguration = options.stamp.compose.deepConfiguration;
+  this.config = Object.freeze(
+    Object.assign({}, configuration, deepConfiguration)
```

Comment:
Typically deep looses to non deep. See stamp-specification. 
Are you sure about deep overwriting non deep configuration?